Shell/Options.cpp

```diff
@@ -2228,7 +2228,8 @@ void Options::init()
                                                                   {"auto","plain","synthesis","off"});
     _questionAnswering.description= "Determines whether (and how) we attempt to answer questions:"
        " plain - answer-literal-based, supports disjunctive answers; synthesis - designed for sythesising programs from proofs.";
-    _questionAnswering.addHardConstraint(If(notEqual(QuestionAnsweringMode::OFF)).then(ProperSaturationAlgorithm()));
+    _questionAnswering.addHardConstraint(If(equal(QuestionAnsweringMode::PLAIN)).then(ProperSaturationAlgorithm()));
+    _questionAnswering.addHardConstraint(If(equal(QuestionAnsweringMode::SYNTHESIS)).then(ProperSaturationAlgorithm()));
```

Comment:
I'm going to regret asking, but shouldn't `If(Or(equal(QuestionAnsweringMode::PLAIN), equal(QuestionAnsweringMode::SYNTHESIS))).then(ProperSaturationAlgorithm()` work?